### ISSUE TITLE
ShopEmailPreviewTask should also allow previewing of OrderStatusLog Emails

### ISSUE THREAD

**Issue by bummzack** [silvershop/silvershop-core#559]
Order Status Log emails were introduced here: https://github.com/silvershop/silvershop-core/pull/534

@anselmdk can you add this?

**Comment by anselmdk** [comment]
Yeah, that was the plan. Thanks for reminding me. I'll look into it.

**Comment by anselmdk** [comment]
So sorry for leaving this around for so long. The truth is, that I'm currently not actively working with SilverShop, so I'll deassign myself.
